console/console-init/ui/src/components/About/About.tsx

```diff
@@ -59,7 +59,9 @@ export const About: React.FunctionComponent<IAboutProps> = ({
   if (docs) {
     docsElement = (
       <>
-        <TextListItem component="dt">Documentation</TextListItem>
+        <TextListItem component="dt" id="about-documentation-dropdown">
```

Comment:
I think we should follow same pattern everywhere so if it's text list item it should be about-documentation-textlistitem

Reply:
Would it be more appropriate to have 'dropdown' at all places instead of 'textlistitem'?

Reply:
I don't really care about that, that is decision what you in UI team have to do. I am just following other ID's which are already using textlistitem :) But if you want know my personal opinion I would go with dropdown in all places :)

Reply:
I'd let this stay textlistitem then, just to make sure we have consistency throughout instead of finding all textlistitem and editing their id's :)

Reply:
@suyash-naithani Please follow the consistency